Resources/translations/FOSUserBundle.da.yml

```diff
@@ -52,11 +52,11 @@ registration:
             Holdet bag siden.
 
 resetting:
-    check_email: |
-        En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.
-        Note: Du kan først bede om en ny adgangskode om %tokenLifetime% timer.
+    check_email: '|
+        En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.'
+    Note: 'Du kan først bede om en ny adgangskode om %tokenLifetime% timer.
```

Comment:
When I try to execute the cache clear command, the application is crashed with the above errors because of the invalid yml. But there are no problems with the validity in the file yml EN.